doc/man3/OPENSSL_init_crypto.pod

```diff
@@ -175,10 +175,10 @@ See OPENSSL_fork_prepare(3) for details.
 
 =item OPENSSL_INIT_NO_ATEXIT
 
-By default OpenSSL will attempt to clean itself up when the process exits via an
-"atexit" handler. Using this option suppresses that behaviour. This means that
-the application will have to clean up OpenSSL explicitly using
-OPENSSL_cleanup().
+Deprecated, it's a no-op currently. Starting version 3.4 OpenSSL
+no longer uses atexit(3) handler to perform its cleanup (OPENSSL_cleanup()).
+The OPENSSL_cleanup() function is now always invoked by library destructor
+function. This is typically arranged by linker.
```

Comment:
the wording I'm using is slightly different, because of issue pointed by @t8m 
```
Disables automatic execution of OPENSSL_cleanup() via library destructor.
Starting from version 3.4 OpenSSL no longer uses atexit(3) handler to call
OPENSSL_cleanup() automatically.  Instead of using atexit(3) the OpenSSL
relies on runtime linker to call OPENSSL_cleanup() via library destructor.
```